### PR TITLE
Make adaptive layout

### PR DESCRIPTION
**Pull request by kudryavtseva99**

### CODE REVIEW

File: img/Vlozheniya_zapreshchennye_k_otpravke_mezhdu_stranami_EAES_Kazahstan.docx
Comment (artur-barmin-ocl) [inline img/Vlozheniya_zapreshchennye_k_otpravke_mezhdu_stranami_EAES_Kazahstan.docx:1]: I guess it shouldn't be in the repository

File: main.js [hunk 1]
```diff
@@ -10,3 +10,18 @@
...
+(function () {
+    const burgerItem = document.querySelector('.burger');
+    const menu = document.querySelector('.header_nav');
+    const menuCloseItem = document.querySelector('.header_nav-close');
+    burgerItem.addEventListener('click', () => {
+      menu.classList.add('header_nav_active');  
+    })
+    menuCloseItem.addEventListener('click', () => {
+        menu.classList.remove('header_nav_active');
+    });
```
Comment (artur-barmin-ocl) [inline main.js:33]: Question for you: can we replace add/remove with toggle? You can read about it [here](https://learn.javascript.ru/styles-and-classes) (just find it with "Find" via CTRL+F)

**Review by artur-barmin-ocl — CHANGES_REQUESTED** [review]

File: img/Vlozheniya_zapreshchennye_k_otpravke_mezhdu_stranami_EAES_Kazahstan.docx
Comment (kudryavtseva99) [reply img/Vlozheniya_zapreshchennye_k_otpravke_mezhdu_stranami_EAES_Kazahstan.docx:1]: Deleted

File: main.js (hunk 1 above)
Comment (kudryavtseva99) [reply main.js:33]: In this case, there is no need to use toggle

**Pull request merged**